pocs/state/states/default/sleeping.py

```diff
@@ -2,7 +2,7 @@ def on_enter(event_data):
     """ """
     pocs = event_data.model
 
-    if pocs.should_retry is False:
+    if pocs.is_safe() and pocs.should_retry is False:
```

Comment:
What happens if safety is fluctuating rapidly? Or can that not happen?

My concern is that we might find things are unsafe at one point during a single transition but later find that things have become safe. This seems to imply that the state machine should only be able to read the safety value once per transition attempt.

Reply:
Once the system is marked unsafe it won't mark safe again for the amount of time specified in `peas.yaml` under `weather.aag_cloud.safety_delay`, which defaults to 15 minutes (on PAN001 I switched it to 30 minutes to handle fluctuating weather).